src/activity_handlers.cpp

```diff
@@ -1530,6 +1530,10 @@ void activity_handlers::oxytorch_finish( player_activity *act, player *p )
         g->m.ter_set( pos, t_window_empty );
         g->m.spawn_item( pos, "steel_plate", rng(0, 1) );
         g->m.spawn_item( pos, "sheet_metal", rng(1, 3) );
+    } else if( ter == t_reb_cage ) {
+        g->m.ter_set( pos, t_pit );
+        g->m.spawn_item( pos, "spike", rng(1, 19) );
+        g->m.spawn_item( pos, "scrap", rng(1, 8) );
```

Comment:
Astyle these two lines, and similar ones in hacksaw below.

Reply:
@Night-Pryanik I'm figuring out how to use the Astyle binary 3.1 that came with Astyle source. The silent failures are a weird experience. Will upload after I figure it out. At least there are manuals.

Reply:
@Night-Pryanik honestly that would waste time and money.